lib/shopping_cart.rb

```diff
@@ -0,0 +1,56 @@
+class ShoppingCart
+  attr_reader :name,
+              :products
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @products = []
+  end
+
+  def capacity
+    @capacity.to_i
+  end
```

Comment:
I like this - my opinion is to always maintain data whenever possible.  However, you will often see devs do this kind of small conversion in the initialize method itself, and then use an attr_reader 🙃